NEWS.md

```diff
@@ -0,0 +1,26 @@
+User-visible changes in ERTP:
+
+## Release v0.1.0 (8/14/2019)
+
+Core ERTP:
+* Refactored contracts to make it easier for users to check the terms of
+  a contract
+* Added an IDL to describe the user-facing ERTP interface for mints,
+  issuers, purses, payments, assays, and the contractHost.
+* Renamed `getExclusive` and `getExclusiveAll` to `claim` and
+  `claimAll`
+* Added `equals` as an assay method
+
+Pixel Demo:
+* Created a new kind of rights for the pixel demo, in which rights are
```

Comment:
Would this be an improvement?

Created a new representation of hierarchical rights for the pixel demo, which allows holders of higher-level rights to revoke rights held further down.

Reply:
I like that a lot better